Curve-finance-suite/Registry-Add-Pool/src/agent.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  Finding,
+  HandleTransaction,
+  FindingSeverity,
+  FindingType,
+} from 'forta-agent';
+
+import  {
+  provideEventCheckerHandler,
+  FindingGenerator,
+} from 'forta-agent-tools';
+
+// The signature of the `PoolAdded` event
+export const ADD_POOL_SIGNATURE = "PoolAdded(address,bytes)";
+
+// The address of the Curve registry from curve.readthedocs.io
+const REGISTRY_ADDRESS: string = '0x90E00ACe148ca3b23Ac1bC8C240C2a7Dd9c2d7f5';
+
+// Creates a finding generator to be used by provideEventCheckerHandler
+const createFindingGenerator = (alertId: string): FindingGenerator =>
+  (metadata: { [key: string]: any } | undefined): Finding => 
+    Finding.fromObject({
+      name: 'Curve Registry contract called',
+      description: 'Event PoolAdded has been emitted',
+      alertId: alertId,
+      severity: FindingSeverity.Info,
+      type: FindingType.Info,
+      protocol: 'Curve Finance',
+      metadata: {
+        // The address of the pool that was added
+        pool_address: metadata!.topics[1],
```

Comment:
`metadata!.topics[1]` is a `bytes32` value, can we decode it as an address for making the finding information more readable?